md/utils.py

```diff
@@ -0,0 +1,32 @@
+from collections import OrderedDict
+
+
+class GroupedData(object):
```

Comment:
Isn't this the same as the class in the NC tree?  You can reference it directly from the NC tree or create a tsutil package with this or something else, but since it isn't state-specific there shouldn't be multiple copies.